apps/gitness/package.json

```diff
@@ -29,6 +29,7 @@
     "@hookform/resolvers": "^3.6.0",
     "@tanstack/react-query": "4.36.1",
     "clipboard-copy": "^3.1.0",
+    "highlight.js": "^11.9.0",
```

Comment:
can you please link the sec approval ticket here?

Reply:
https://harness.atlassian.net/browse/SECAPP-1451